src/Zio/FileSystems/AggregateFileSystem.cs

```diff
@@ -287,7 +287,11 @@ protected override Stream OpenFileImpl(UPath path, FileMode mode, FileAccess acc
     protected override FileAttributes GetAttributesImpl(UPath path)
     {
         var entry = GetPath(path);
-        return entry.FileSystem.GetAttributes(path) | FileAttributes.ReadOnly;
+        var attributes = entry.FileSystem.GetAttributes(path);
+
+        return attributes == FileAttributes.Normal
+            ? FileAttributes.ReadOnly
```

Comment:
Breaking change: Normal is used when there are no attributes. Combining "Normal" and "ReadOnly" isn't valid, so I changed it so it only returns ReadOnly.